### ISSUE TITLE
Document existing what PowerShell support exists and how to use it

### ISSUE THREAD

**Issue by leezen** [pulumi/docs#11783]
Given the interest in https://github.com/pulumi/pulumi/issues/3507 we should document what you can already do with PS (and any potential issues you might run into).

**Comment by thoward** [comment]
This is a great idea, and it seems like interest in Powershell support remains strong (per continued user commentary on https://github.com/pulumi/pulumi/issues/3507). However, I think it's prudent to wait to include it in our official documentation until it's fully implemented/supported. This might make a good blog post to feature community contributions.

Closing this for now as `wont-fix`. We can reopen or file a new ticket for a blog (if we want to show what's currently available) or docs (once the language support is complete).
